server/views/pages/view/cases.njk

```diff
@@ -155,17 +164,40 @@
                     <h2 class="govuk-heading-l">Future releases</h2>
                     <p class="govuk-body">Licences for people within 4 weeks of release or that have been started before then.</p>
-                    {{ viewCases({ cases: cases, isHardStop: false, config: statusConfig}) }}
+                    {{ viewCases({ cases: cases, tabType: 'futureReleases', config: statusConfig}) }}
                     <div aria-live="polite" role="status" aria-atomic="true" class="govuk-visually-hidden"></div>
                 </div>
 
+                {% if (cases | tabType('attentionNeeded')).length %}
+                    <div class="govuk-tabs__panel govuk-tabs__panel--hidden" id="attention-needed" role="tabpanel" aria-labelledby="tab_attention-needed">
+                        <h2 class="govuk-heading-l">Attention needed</h2>
+                        <p class="govuk-body">Check the legal status of people below to see if you need to request support. 
+                            <br><strong>You must not release someone in this list with the licence created in this service</strong>.</p>
+                            {{ govukDetails({
+                                summaryText: "Why are these licences showing here?",
+                                html: "<p class='govuk-body'>Create and vary a licence gets key dates from NOMIS. If key dates are removed from NOMIS, <br> the licence will have no dates and remain in the list of people in prison.</p> <p class='govuk-body'>This means the probation practitioner cannot see or work on the licence.</p>",
+                                classes: ""
+                            }) }}
+                            <h3 class="govuk-heading-m" style="padding-top: 10px;">If legal status is remand or convicted unsentenced</h3>
+                            <ul class="govuk-list govuk-list--bullet">
+                                <li>If their charges are dropped, they are found not guilty or get a non-custodial sentence, contact
+                                    &nbsp;createandvaryalicence@digital.justice.gov.uk&nbsp;for support immediately</li>
+                                <li>If someone gets a new custodial sentence, then you do not need to do anything</li>
+                            </ul>
+                            <h3 class="govuk-heading-m">Any other legal status</h3>
+                            <p class="govuk-body" style="padding-bottom: 20px;">Contact&nbsp;createandvaryalicence@digital.justice.gov.uk for support immediately.
+                                <br> For example, the legal status could be immigration detainee.</p>
+                            {{ viewCases({ cases: cases, tabType: 'attentionNeeded', config: statusConfig}) }}
+                        <div aria-live="polite" role="status" aria-atomic="true" class="govuk-visually-hidden"></div>
```

Comment:
whats this div for?

Reply:
Just cloned the other tab and forgot to remove it.
Now I have removed unwanted div for all the other's as well.